splinterd/src/config/clap.rs

```diff
@@ -204,7 +204,7 @@ mod tests {
             (@arg tls_server_cert: --("tls-server-cert") +takes_value)
             (@arg tls_server_key:  --("tls-server-key") +takes_value)
             (@arg tls_client_key:  --("tls-client-key") +takes_value)
-            (@arg bind: --("bind") +takes_value)
+            (@arg rest_api_endpoint: --("rest-api-endpoint") +takes_value)
```

Comment:
Please add an alias for `--bind` so both will work

Reply:
Added